### PR TITLE
Fix downgrade script

### PR DESCRIPTION
**Pull request by andrewoh531**

### CODE REVIEW

File: api/migrations/versions/0199_change_provider_details.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def make_provider_details_downgrade_sql(identifier):
             ORDER BY version desc
             LIMIT 1
         ) h
-        WHERE identifier = '{0}'
+        WHERE provider_details.identifier = '{0}'
```
Comment (jonathaningram) [inline api/migrations/versions/0199_change_provider_details.py]: @andrewoh531 is this required?

File: api/migrations/versions/0199_change_provider_details.py [hunk 2]
```diff
@@ -35,7 +35,7 @@ def make_provider_details_history_upgrade_sql(identifier):
 def make_provider_details_history_downgrade_sql(identifier):
     return """
         DELETE FROM provider_details_history
-        WHERE identifier = {0} AND version = (SELECT version from provider_details WHERE identifier = '{0}')
+        WHERE identifier = '{0}' AND version = (SELECT version from provider_details WHERE identifier = '{0}')
```
Comment (jonathaningram) [inline api/migrations/versions/0199_change_provider_details.py]: @andrewoh531 thanks.

File: api/migrations/versions/0199_change_provider_details.py (hunk 1 above)
Comment (dagoof) [reply api/migrations/versions/0199_change_provider_details.py]: sql complaints that it is ambiguous otherwise

Comment (jonathaningram) [reply api/migrations/versions/0199_change_provider_details.py]: Cool, thanks for fixing my rubbish SQL.

Comment (andrewoh531) [reply api/migrations/versions/0199_change_provider_details.py]: Yeah it was complaining of ambiguous column name (line number in the logs doesn't match up to the file):
```
sqlalchemy.exc.ProgrammingError: (psycopg2.ProgrammingError) column reference "identifier" is ambiguous
LINE 19:         WHERE identifier = 'firetext'
```

**Pull request merged**